### DISCUSSION TITLE
Welcome to the Tanzu Community Edition Community!

### DISCUSSION THREAD

**Discussion by pnbrown** [vmware-tanzu/community-edition#2101] [General]
Hello!

I am excited to welcome you to the Tanzu Community Edition discussion board on GitHub. This community is here for you. We want you to feel empowered to learn, ask questions, contribute, and help each other out. To that end, we have our [Code of Conduct](https://github.com/vmware-tanzu/community-edition/blob/main/CODE_OF_CONDUCT.md) which essentially charges us to be excellent to one another.

Please feel free to introduce yourselves to us and meet other community members in Introductions thread.
 
We are incredibly happy to have you on this journey with us. Thank you for being here.

-Nigel Brown, Community Manager for Tanzu Community Edition

**Comment by jpmcb** [comment]
Woot woot! 🥳 Welcome all - very excited to get started and introduce the community to Tanzu Community Edition!
